src/background/index.js

```diff
@@ -1,30 +1,58 @@
-import { TEXT_SCAN } from '../extensionMessageTypes';
+import { LOOKUP_PLACE, SCAN_PARAGRAPH } from '../extensionMessageTypes';
 
-function selectionTextListener(info) {
+function lookupPlaceListener(info) {
```

Comment:
you can do object deconstruction here... (find info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)) as it is actually prefered when the object your passing only has a few parameters that you are using in the function. Namely, this line would become:
```function lookupPlaceListener({ selectionText }) {```